### ISSUE TITLE
Focus trap on mobile devices

### ISSUE THREAD

**Issue by ccachia** [jackocnr/intl-tel-input#1346]
When trying out the component for accessibility on both desktop and mobile devices, we identified that focus trap on mobile devices does not work.  This can be replicated from the demo page.

### Steps to reproduce
1.  Navigate to https://intl-tel-input.com/ from a mobile device
2.  Activate Talk Back or similar
3.  Double tap on the flags icon to open the dropdown
4. Swipe to navigate through the list of countries until reaching the last country

### Expected behaviour
When the last country in the list is in focus and the user swipes to navigate to the next country (which doesn't not exist), the focus should go to either the first item in the list or else remain on the last item in the list

### Actual behaviour
When the last country in the list is in focus and the user swipes to navigate to the next country (which doesn't not exist), the popup remains open yet focus moves to an element on the page

**Comment by jackocnr** [comment]
Apologies for the delay in getting to this. Sometimes life gets in the way!

I can reproduce this issue, but I would have no idea how to go about fixing it.

Our aria tags are based on [this example](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/), and annoyingly if I try to navigate that page with Talk Back, in the example section, I can open the dropdown but it wont even let me select a list item! So it's completely broken. At least with intl-tel-input you can scroll through the list and select an item!

So I'm not sure how to proceed here. If anyone has any ideas please do share them.
